tree/ntuple/inc/ROOT/REntry.hxx

```diff
@@ -45,7 +45,7 @@ class RNTupleJoinProcessor;
 /**
 \class ROOT::REntry
 \ingroup NTuple
-\brief The REntry is a collection of values in an ntuple corresponding to a complete row in the data set.
+\brief The REntry is a collection of values in an RNTuple corresponding to a complete row in the data set.
```

Comment:
Does the next line (`The memory locations that are associated with values are managed through shared pointers.`) need to be updated when taking in consideration the raw bindings?

Is clear from other context (am I even right ;)) that 'data set` means the data in the (potentially sub-)set of fields described by the model used to create the `REntry` (I assume it is clear).

Reply:
Raw pointers are internally encapsulated in a (non-owning) shared pointer.

I don't fully understand your second point, is it about imposing a model to read only a subset of the dataset and the nuance that the `REntry` is then not a complete row of the dataset on disk? We can maybe word this differently, but I don't see this as related to the notes about type checking.

Reply:
>  but I don't see this as related to the notes about type checking.

It is not.  It is indeed 'just' about the semantic of `complete` and/or `data set` in `complete row in the data set` (i.e. yes it is "about imposing a model to read only a subset of the dataset and the nuance that the REntry is then not a complete row of the dataset on disk").

Reply:
Yes since I can not think of succinct way of making the statement clearer ....